runtime/common/src/test/kotlin/kotlinx/serialization/json/JsonTestBase.kt

```diff
@@ -15,6 +15,22 @@ abstract class JsonTestBase {
     protected val unquoted = Json(unquoted = true)
     protected val nonstrict = Json(strictMode = false)
 
+    init {
+        strict.register()
+        unquoted.register()
+        nonstrict.register()
+    }
+
+    private fun Json.register() {
+        val mutableCtx = (context as MutableSerialContext)
```

Comment:
This is an implementation detail. Are you using it intentionally? Otherwise, please use `install` (maybe some shorthand builder worth implementing and adding to the library)